test/cases/api.sh

```diff
@@ -193,9 +193,6 @@ function dump_db() {
   # Disable -x for these commands to avoid printing the whole result and manifest into the log
   set +x
 
-  # Make sure we get 3 job entries in the db per compose (depsolve + manifest + build)
-  sudo ${CONTAINER_RUNTIME} exec "${DB_CONTAINER_NAME}" psql -U postgres -d osbuildcomposer -c "SELECT * FROM jobs;" | grep "9 rows" > /dev/null
-
```

Comment:
Can this be replaced with the expected number or is it not fixed anymore because sometimes there's an ostree resolve job?  Could we make it conditional?

Reply:
So I wonder how useful this check was. Do we really care about how many jobs it takes to get to the end result?

Also for ostree jobs it's now 4 ; and for non-ostree it's 3.